Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -470,9 +521,16 @@ def shuffle_split_changed(self):
         self._param_changed()
 
     def _param_changed(self):
+        self.modcompbox.setEnabled(self.resampling == OWTestLearners.KFold)
+        self._update_comparison_enabled()
         self._invalidate()
         self.__update()
 
+    def _update_comparison_enabled(self):
+        self.comparison_table.setEnabled(
```

Comment:
I like disabling it because it shows the user that it's intentionally blank. Otherwise it looks like a bug when the upper table is filled and this one isn't (e.g. when using Leave one out). Hiding would also be an option, though I like disabling better -- like "something could be here, but currently isn't because I can't compute it in this situation".

I can disable it when there is no data. But in this case we should do the same with the above table, I suppose. We need to discuss this.

Reply:
Both views now disable under same conditions.